classes/class-wc-connect-api-client.php

```diff
@@ -466,7 +466,7 @@ protected function request_body( $initial_body = array() ) {
 					'dimension_unit'       => strtolower( get_option( 'woocommerce_dimension_unit' ) ),
 					'weight_unit'          => strtolower( get_option( 'woocommerce_weight_unit' ) ),
 					'wcs_version'          => WC_Connect_Loader::get_wcs_version(),
-					'jetpack_version'      => JETPACK__VERSION,
+					'jetpack_version'      => defined( 'JETPACK__VERSION' ) ? JETPACK__VERSION : 'embed',
```

Comment:
I think it was only ever used for troubleshooting. If it's easy to get the version of the `jetpack-connection` package I would include it, otherwise `embed` is fine.

Reply:
Yep, just checked and we are not using it anywhere [aside from Tracks calls](https://github.com/Automattic/woocommerce-connect-server/blob/32608bf9fffe39abaaa40826d3d06c537948c086/lib/analytics/tracks.js#L73). We [require it to be a string](https://github.com/Automattic/woocommerce-connect-server/blob/32608bf9fffe39abaaa40826d3d06c537948c086/lib/settings/validator.js#L18) **if** it is present.